proxy/handler/authorize_handler.go

```diff
@@ -0,0 +1,137 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package handler
+
+import (
+	"bytes"
+	"encoding/base64"
+	"errors"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/apache/servicecomb-mesher/proxy/config"
+	"github.com/apache/servicecomb-mesher/proxy/plugins/authorize"
+	"github.com/go-chassis/go-chassis/core/handler"
+	"github.com/go-chassis/go-chassis/core/invocation"
+)
+
+//Authorize is a constant
+const Authorize = "authorize"
+
+//AuthHandler ..
+type AuthHandler struct {
+	config *config.Authorize
+}
+
+//Handle is provider
+func (au *AuthHandler) Handle(chain *handler.Chain, inv *invocation.Invocation, cb invocation.ResponseCallBack) {
+	au.config = config.GetConfig().Authorize
+	if au.config == nil {
+		log.Println("authorize config not found")
```

Comment:
Please use the same log tool "openlogging".

Reply:
Ok! Already modified.